Formula/cryptol.rb

```diff
@@ -4,9 +4,9 @@ class Cryptol < Formula
   include Language::Haskell::Cabal
 
   desc "Domain-specific language for specifying cryptographic algorithms"
-  homepage "https://www.cryptol.net/"
-  url "https://hackage.haskell.org/package/cryptol-2.6.0/cryptol-2.6.0.tar.gz"
-  sha256 "5f8abbfa2765ac0f6bb887edbec7032677d107c39581a4c78614e97382738f42"
+  homepage "https://github.com/GaloisInc/cryptol"
```

Comment:
`https://www.cryptol.net/` is not reachable.

Reply:
Maybe they had an issue recently but it's back up. I think we should keep it as homepage.

Reply:
I will revert this line change.